### PR TITLE
Updated Functions v2.0.1

### PR DESCRIPTION
**Pull request by Ifera**
## Description
* Updated api for latest pockeetmine-mp release.
* Fixed command not being removed.

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Functions
 author: BoxOfDevs Team
-version: 2.0.0
-api: [3.0.0-ALPHA12, 3.0.0-ALPHA13, 3.0.0-ALPHA14]
```
Comment (HimbeersaftLP) [inline plugin.yml:4]: ~~Why remove all those? You didn't change anything that breaks bc.~~
See [new comment](https://github.com/BoxOfDevs/Functions/pull/17#issuecomment-408664390)

**Review by HimbeersaftLP — CHANGES_REQUESTED** [review]
~~Seems to be working for me, but I see no reason to remove all those previous API versions.~~
See [new comment](https://github.com/BoxOfDevs/Functions/pull/17#issuecomment-408664390)

**Review by Ifera — COMMENTED** [review]
Since the commands were not being removed before hand due to a typo, [this](https://github.com/BoxOfDevs/Functions/pull/17/commits/a7c706b4f2f5c6d03661e17dcd4ad4477180a235) to be exact

**Comment by HimbeersaftLP** [comment]
~~I'm not talking about the plugin version, I mean the API version.~~
See [new comment](https://github.com/BoxOfDevs/Functions/pull/17#issuecomment-408664390)

File: plugin.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: Functions
 author: BoxOfDevs Team
 version: 2.0.1
-api: [3.0.0]
+api: [3.0.0-ALPHA12, 3.0.0]
```
Comment (HimbeersaftLP) [inline plugin.yml]: I swear there were more of them before, considering we advertise that compatibility in the README and that it's supported, I see no reason not to add them

**Review by HimbeersaftLP — APPROVED** [review]

**Comment by HimbeersaftLP** [comment]
Sorry for my stupidness, I didn't read the numbers and thought those were older ones, I wonder who added them though, as that was a really bad idea.

**Review by Ad5001 — APPROVED** [review]
Seems good to me.

**Review by lavender-sys — APPROVED** [review]

**Pull request merged**